flavors/liquid-mainnet/extras.css

```diff
@@ -85,7 +85,7 @@
 .sortable{
   cursor: pointer;
 }
-.sortable::after{
+.desc::after{
```

Comment:
What is `.desc`? I can't seem to find this class being used anywhere in the html

Reply:
const reverseSortDir = encodeURIComponent(sort_dir === 'asc' ? 'desc' : 'asc')

**html :**  <a href={`assets?sort_field=name&sort_dir=${reverseSortDir}`} 
                  className={`assets-table-cell sortable ${sort_field === "name" ? sort_dir : ""}`}>{t`Name`}
                </a>

Reply:
My bad -- I was looking at the wrong deployment so didn't see it.